webapp/src/pages/Settings.tsx

```diff
@@ -746,7 +747,65 @@ const Settings: React.FC<Props> = ({ onClose }) => {
           ) : Array.isArray(value) ? (
             displayArgumentsList(field, value)
           ) : typeof value === "object" ? (
-            displayKeywordArguments(field, value)
+            <Box
+              key={field}
+              display="flex"
+              flexDirection="column"
+              marginRight={2}
+            >
```

Comment:
Let's not add extra margin here, we need as much place as possible for the text.
```suggestion
            <Box key={field} display="flex" flexDirection="column">
```

Reply:
Ohh!.. Right. I added to test the column gap but. I forgot to remove it later. thanks for pointing it out.

Reply:
Looks like you accidentally reverted your fix when you rebased and [force-pushed](https://github.com/ServiceNow/azimuth/compare/720b8aab50e70e2f8b3f87457758bcdf779e6219..d16611e32c832dfa566d95b6c9b83be9278f99a0#diff-2ee47bc0fd1761ec5486a838d588adc51eb53c7b901fd5c3a6c1ad24ba6f11b2L749-R956).

Reply:
Precision, it seems you lost the following two commits. You could cherry-pick them to fix the issue.
- d6e167df236d83f1824b73c1696b6397e64eb630
- 720b8aab50e70e2f8b3f87457758bcdf779e6219